blocks/api/post.pegjs

```diff
@@ -1,15 +1,11 @@
 {
 
-function untransformValue( value ) {
-	return 'string' === typeof value
-		? value.replace( /\\-/g, '-' )
-		: value;
-}
-
-function keyValue( key, value ) {
-  const o = {};
-  o[ key ] = untransformValue( value );
-  return o;
+function maybeJSON( s ) {
```

Comment:
right now the error is reported as a `null` value. many things, including the string `null` is valid JSON. in that sense, I would like to propose that we _only_ accept JSON _objects_, which would dramatically simplify the parser and semantics and serializer.

that would mean `null` can also signify an error, or alternatively, on error we could send the string of the content which failed to parse.